server/assets/embed.go

```diff
@@ -67,3 +66,17 @@ func fromPath(path string) (assets fs.FS, err error) {
 
 	return
 }
+
+func FileNames(aPath string) (fileNames []string) {
+	files, err := fs.ReadDir(ff, "src/"+aPath)
```

Comment:
This will only return names in the given directory; if you need to do a recursive job, you'll need to add another loop into this bad boy